ejb3/src/main/java/org/jboss/as/ejb3/component/EJBViewDescription.java

```diff
@@ -36,7 +36,8 @@
 import org.jboss.invocation.proxy.ProxyFactory;
 import org.jboss.metadata.ejb.spec.MethodInterfaceType;
 import org.jboss.msc.service.ServiceName;
-import org.jboss.msc.value.Value;
+
+import java.util.function.Supplier;
 
```

Comment:
can we move the java.* import to the top, which I think is the convention in WF?

Reply:
Fixed